config/locales/en/settings.yml

```diff
@@ -132,7 +132,7 @@ en:
         method_name: "Request method name"
         method_name_description: "Request method name accepted by the City Census WebService."
         structure: "Request Structure"
-        structure_description: 'Request Structure that receives the Census WebService of the City council. The "static" values of this request should be filled. Values related to Document Type, Document Number, Date of Birth and Postal Code should be blank.'
+        structure_description: 'Request Structure that receives the Census WebService of the City council. The "static" values of this request should be filled. Values related to Document Type, Document Number, Date of Birth and Postal Code should be null. Example of a valid format: { "request": { "codigo_institucion": 1, "codigo_portal": 1, "codigo_usuario": 1, "documento": null, "tipo_documento": null, "codigo_idioma": 102, "nivel": 3 } }'
```

Comment:
Does it make sense to use English names in the parameters here? I've no idea; just asking :smile:.

Reply:
Nice catch @javierm 
But I did it consciously. The reason is because I have copied the request of CensusAPI so that it can be more easily understood what is intended to be done with these fields.
But I can change it to English if you don't think that's motive enough.

Reply:
Leaving it like this is fine with me :+1:.